src/preflightCheck.js

```diff
@@ -54,7 +54,8 @@ export default function createPreflightCheck () {
 				ctx.error( MODULE_ERROR );
 			}
 
-			if ( ~check.indexOf( '/helpers/inherits' ) ) helpers = RUNTIME;
+      console.log(check)
+      if ( check.match( /\/helpers\/(builtin\/)?(es6\/)?inherits/ ) ) helpers = RUNTIME;
```

Comment:
also there is some indentation problem here - I guess we need to start using prettier, gonna prepare a PR for it later, but for now it should be changed manually

Reply:
Fixed